x/campaign/types/message_add_vesting_options.go

```diff
@@ -0,0 +1,66 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+var _ sdk.Msg = &MsgAddVestingOptions{}
+
+func NewMsgAddVestingOptions(
+	campaignID uint64,
+	coordinator,
+	address string,
+	shares Shares,
```

Comment:
I think it's a mistake from the `VestingAccount` initialization. We should rather name this field `StartingShares` to precise those are the shares held when the chain start